src/PkgCommitPage.cc

```diff
@@ -363,7 +363,8 @@ void PkgCommitPage::initProgressData()
     _completedInstalledSize = 0;
     _completedTasksCount    = 0;
 
-    foreach ( PkgTask * task, pkgTasks()->todo() )
+    const PkgTaskList tasks = pkgTasks()->todo();
+    for ( const PkgTask * task : tasks )
```

Comment:
Please add a blank line before the `for()`.

Reply:
On second thought, please let's leave that as the one-liner that it was; creating another variable that the next line refers to (and then is never used again) adds to the mental burden of reading the code.

And as I wrote elsewhere here, the compiler will generate the exact same code; "common subexpression" optimization is the simplest one there is, one that every compiler always does. For sure the functions will not be called several times; alone for the fact that there might be unexpected side effects when doing that.

Reply:
Added blank line. Keeping the local variable for correctness, not performance.